modules/ROOT/pages/api-glossary.adoc

```diff
@@ -1,15 +1,15 @@
 = Bonita REST API glossary
 :description: List of Bonita REST API definitions and descriptions.
 
-List of Bonita REST API definitions and descriptions.
+{description}
 
 == Definition
 
 *Definition*: in Bonita Studio you create definition of processes. A definition of a process includes name, version, definition of actors name... +
```

Comment:
In Bonita Studio you create definition of processes. => In Bonita Studio you create processes definition. 
A definition of a process includes name => The definition of a process includes name